src/content/blog/astro-480.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "Astro 4.8"
+description: "Astro 4.8 is here! This release includes experimental support for Astro actions and rewriting, performance improvements, and more."
+publishDate: "May 9, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+  - sarah
+coverImage: "/src/content/blog/_images/astro-480/header_astro_4_8.webp"
+socialImage: "/src/content/blog/_images/astro-480/og_image_4_8.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+Astro 4.8 is out now! This release includes experimental support for Astro actions and rewriting, performance improvements, and more.
+</BlogSummary>
+
+Full release highlights include:
+
+- [**Experimental: Astro Actions**](#experimental-astro-actions)
+- [**Experimental: Rewriting support**](#experimental-rewriting-support)
+- [**Performance improvements**](#performance-improvements)
+- [**Ability to define multiple routes with the same entrypoint**](#ability-to-define-multiple-routes-with-the-same-entrypoint)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental: Astro Actions
```

Comment:
No real preference here, but did you decide on calling them "Astro actions"?  I notice that we don't say "Actions API" here anywhere, which is one way we can be referring to the feature (with a capital A!) if we need to vary language use so we're not saying the same thing over and over again.

Reply:
Ah, I don't think we want to use Actions API! That is easily confused with API endpoints. Astro Actions or simply Actions were the two ways we intended for docs.